docs/Privacy/How-To/Pantheon-Privacy.md

```diff
@@ -0,0 +1,49 @@
+description: Pantheon-extended privacy
+<!--- END of page meta data -->
+
+# Using Pantheon-extended Privacy 
+
+Pantheon provides an extended implementation of privacy. When using Pantheon-extended privacy, a [privacy
```

Comment:
Minor suggestion:
Pantheon provides an extended privacy implementation by allowing a privacy group to be created for a set of participants with a unique privacy group ID. The privacy group ID must be specified when sending private transactions.

Reply:
Fixed